packages/styles/scss/patterns/listsection/_listsection-group.scss

```diff
@@ -6,16 +6,14 @@
 //
 
 @mixin listsection-group {
-  .#{$prefix}--listsection-group {
-    &__title {
-      @include type-style('expressive-heading-04');
+  .#{$prefix}--content-group__title {
+    padding-left: 0;
```

Comment:
This shouldn't be necessary... Looking at the `ContentGroup` it looks as if there are still some grid stuff in there?

Reply:
I removed it and looks to be adding some padding to the left of the title by default. I say let's leave this in here for now, and come back to it when we refactor the MixedArray.